scripts/create-clusters.sh

```diff
@@ -38,12 +38,15 @@ function create-insecure-registry() {
 function configure-insecure-registry() {
   local err=
   if sudo test -f "${docker_daemon_config}"; then
-    echo <<EOF "Error: ${docker_daemon_config} exists and \
-CONFIGURE_INSECURE_REGISTRY=${CONFIGURE_INSECURE_REGISTRY}. This script needs \
-to add an 'insecure-registries' entry with host '${CONTAINER_REGISTRY_HOST}' to \
-${docker_daemon_config}. Please make the necessary changes or backup and try again."
+    if sudo grep -q "\"insecure-registries\": \[\"${CONTAINER_REGISTRY_HOST}\"\]" ${docker_daemon_config}; then
+      return 0
+    else
+      echo <<EOF "Error: ${docker_daemon_config} exists and \
+is already configured with an 'insecure-registries' entry but not set to ${CONTAINER_REGISTRY_HOST}. \
```

Comment:
A small problem with the error message now. This branch may also be entered when there is no 'insecure-registries' entry.

Reply:
done.